snuba/datasets/discover.py

```diff
@@ -72,15 +72,9 @@
 logger = logging.getLogger(__name__)
 
 EVENT_CONDITION = FunctionCallMatch(
-    None,
     Param("function", Or([StringMatch(op) for op in BINARY_OPERATORS])),
     (
-        Or(
-            [
-                ColumnMatch(None, None, StringMatch("type")),
-                LiteralMatch(StringMatch("type"), None),
```

Comment:
@evanh  do you know why we were trying to match an alias here ? It is the only palce in the code base.

Reply:
Sentry tests are fine